### PR TITLE
create Tooltip

### PR DESCRIPTION
**Pull request by Maxwell929**

### CODE REVIEW

File: components/ProjectCard.tsx [hunk 1]
```diff
@@ -11,16 +11,11 @@ export const ProjectCard = ({
...
-                iconClassName="w-6 h-6 text-gray-700  xl:hover:scale-110 hover:text-gray-500"
-            />
-            <p className="absolute top-0 left-0 z-10 p-2 font-bold">{year}</p>
-            <div className="absolute inset-0 top-0 h-[6rem] bg-gray-300"></div>
+        <article className="relative z-10 min-h-[20rem] w-[18rem] rounded-lg bg-white p-4 duration-300 md:hover:scale-110">
+            <p className="absolute top-0 left-0 z-10  p-2 font-bold">{year}</p>
+            <div className="absolute inset-0 top-0 h-[6rem] rounded-t-lg bg-gray-300"></div>
 
-            <div className="flex flex-col items-center gap-4">
+            <div className=" flex flex-col items-center gap-4">
```
Comment (WieFel) [inline components/ProjectCard.tsx]: not needed leading space in className

File: components/Icon.tsx [hunk 2]
```diff
@@ -6,16 +6,32 @@ export interface IconProps {
...
     return (
         <a
-            className={url === undefined ? 'pointer-events-none' : ''}
+            className={
+                `group relative ` +
+                (url === undefined ? 'pointer-events-none' : '')
+                // ' ' +
+                // (url != null
+                //     ? 'after:absolute after:top-8 after:left-1/2 after:z-20 after:flex after:-translate-x-1/2 after:rounded-lg after:bg-gray-700 after:px-4 after:py-2 after:text-sm after:text-white after:opacity-0 after:duration-300 after:content-["githubandGitlab"] after:hover:opacity-100'
+                //     : '')
```
Comment (WieFel) [inline components/Icon.tsx]: - remove commented code
- ` can be replaced with normal " if no string interpolation is used

File: components/Icon.tsx [hunk 3]
```diff
@@ -6,16 +6,32 @@ export interface IconProps {
...
+            }
             href={url}
             target={url === undefined ? '' : '_blank'}
             rel="noreferrer"
         >
+            {tooltip != null ? (
+                <div className="pointer-events-none absolute top-[calc(100%+.5rem)] left-1/2 z-20 hidden w-28 -translate-x-1/2 rounded-lg bg-gray-700 p-2 text-center text-[.7rem] text-sm text-white opacity-0 duration-300 group-hover:opacity-100 xl:block">
```
Comment (WieFel) [inline components/Icon.tsx]: Create a simple component for `Tooltip`!?

**Review by WieFel — CHANGES_REQUESTED** [review]

File: components/Icon.tsx (hunk 3 above)
Comment (Maxwell929) [reply components/Icon.tsx]: @WieFel will do that as quick as I can

**Review by WieFel — APPROVED** [review]

**Pull request merged**